examples/listen/main.go

```diff
@@ -25,7 +25,7 @@ func main() {
 	//
 
 	// Prepare the configuration of the DTLS connection
-	config := &dtls.Config{Certificate: certificate, PrivateKey: privateKey}
+	config := &dtls.Config{Certificate: certificate, PrivateKey: privateKey, PSK: []byte{0x00, 0x01}}
```

Comment:
Wouldn't this result in an error since both a certificate and a PSK are specified?

Reply:
Oops nice catch, accident on my part when testing :/